### PR TITLE
pyreverse: Add output directory command-line option

### LINKED ISSUE #4159: Pyreverse output directory
Could pyreverse (included with pylint) please have an option to choose an output directory?

I wrote a script that auto-generates UML diagrams for each .py file in my package, but then I have to copy them one by one with shutil.move(), because you don't have an option to specify the output path in pyreverse. (Only the output format).

Also I'm not the only one who needs this feature:
https://stackoverflow.com/questions/48689311/pyreverse-how-to-set-output-directory-for-generated-diagrams

### PR DESCRIPTION
**Pull request by mbyrnepr2**
Issue: #4159

## Steps

- [ ] Add yourself to CONTRIBUTORS if you are a new contributor.
- [ ] Add a ChangeLog entry describing what your PR does.
- [ ] If it's a new feature or an important bug fix, add a What's New entry in `doc/whatsnew/<current release.rst>`.
- [ ] Write a good description on what the PR does.

## Type of Changes

|   | Type |
| ------------- | ------------- |
| ✓  | :bug: Bug fix  |

### CODE REVIEW

**Comment by matusvalo** [comment]
black pre-commit hook failed. @mbyrnepr2 running black over codebase should fix the failing CI.

File: ChangeLog [hunk 1]
```diff
@@ -17,6 +17,10 @@ Release date: TBA
 ..
   Put bug fixes that will be cherry-picked to latest major version here
 
+* Introduce a command-line option to specify pyreverse output directory
```
Comment (Pierre-Sassoulas) [inline ChangeLog:20]: It's a feature, but it's small and convenient, good call on putting it in 2.7.

File: pylint/pyreverse/writer.py [hunk 2]
```diff
@@ -32,6 +34,10 @@ def write(self, diadefs):
         for diagram in diadefs:
             basename = diagram.title.strip().replace(" ", "_")
             file_name = f"{basename}.{self.config.output_format}"
+            if os.path.exists(self.config.output_directory):
+                file_name = os.path.join(
```
Comment (Pierre-Sassoulas) [inline pylint/pyreverse/writer.py:38]: You can auto-format the code by launching ``pre-commit run --all-files``

**Review by Pierre-Sassoulas — APPROVED** [review]

**Comment by Pierre-Sassoulas** [comment]
Thank you for this handy new feature, and congratulation on becoming a pylint contributor ! This will be released in pylint `2.7.3` probably today.

**Pull request merged**